src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/ReleasePrisonerService.kt

```diff
@@ -29,8 +29,8 @@ class ReleasePrisonerService(
   private val activityMovementService: ActivityMovementService,
 ) {
 
-  fun releasePrisoner(prisonerIdentifier: String, request: RequestToReleasePrisoner): InmateDetail {
-    val booking = getOffenderBooking(prisonerIdentifier).flatMap { it.isActiveIn() }.getOrThrow()
+  fun releasePrisoner(prisonerIdentifier: String, request: RequestToReleasePrisoner, mustBeActiveIn: Boolean = true): InmateDetail {
+    val booking = getOffenderBooking(prisonerIdentifier).flatMap { if (mustBeActiveIn) it.isActiveIn() else success(it) }.getOrThrow()
```

Comment:
Keep the workaround to allow released patients to released again - this time to hospital. Ugh.